ez-launch/sources/ez_launch.move

```diff
@@ -234,7 +219,40 @@ module ez_launch::ez_launch {
         collection
     }
 
-    public fun pre_mint_token_helper(
+    public fun pre_mint_tokens_helper(
+        creator: &signer,
+        collection: Object<Collection>,
+        token_name_vec: vector<String>,
+        token_uri_vec: vector<String>,
+        token_description_vec: vector<String>,
+        num_tokens: u64,
+    ) acquires CollectionDetails, EZLaunchRefs {
+        assert_owner(signer::address_of(creator), collection);
```

Comment:
The line needed here? ` let object_signer = authorized_collection_signer(creator, collection);` does this for us?